test_unstructured/partition/csv/test_tsv.py

```diff
@@ -8,9 +8,9 @@
 )
 from unstructured.cleaners.core import clean_extra_whitespace
 from unstructured.documents.elements import Table
-from unstructured.partition.json import partition_json
 from unstructured.partition.tsv import partition_tsv
-from unstructured.staging.base import elements_to_json
+
+from ...unitutils import assert_round_trips_through_JSON, example_doc_path
```

Comment:
please avoid relative imports

Reply:
I do. This is necessary because this is in the tests and the tests are not an installed module so have no qualified name. This module would never appear in the library code nor would we need relative imports..

Reply:
@cragwolfe I fixed this, I missed that we were already importing things from `test_unstructured/...`. Note that this only works when tests are run from the root directory of the repo, but that seems to not be a problem so far.